html/template/admin/assets/js/function.js

```diff
@@ -222,4 +219,31 @@ $(function () {
         $('body').append($form); // Firefox requires form to be on the page to allow submission
         $form.submit();
     });
+
+    // scroll to error message if have
+    $(window).load(function() {
+        var el = $(".errormsg");
+        if (el.length) {
+            // Open panel when has error
+            openPanel(el.first());
+            setTimeout(function () {
```

Comment:
On the phone, the interface render very slow, so it has to wait for the interface loaded.